src/ccs/mod.rs

```diff
@@ -47,7 +48,7 @@ mod util;
 /// As well as m, n, s, s_prime
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(bound = "")]
-pub struct CCSShape<G: Group> {
+pub struct CCS<G: Group> {
```

Comment:
Same here as above, this doesn't seem like an improvement to me. If it is due to some semantic baggage with "Shape" (not sure what that'd be) maybe some other qualifier can be used?

Reply:
Same reply as above. But if there are strong opinions over this. I can change it.